arches_lingo/serializers.py

```diff
@@ -63,30 +64,37 @@ def validate_appellative_status(self, data):
             raise ValidationError(msg)
 
         if data:
```

Comment:
not blocking on this, but this function looks like it could be broken out into other functions?